app/src/main/java/com/fstyle/androidtrainning/screen/main/playingfragments/lyrics/LyricsFragment.java

```diff
@@ -1,23 +1,52 @@
 package com.fstyle.androidtrainning.screen.main.playingfragments.lyrics;
 
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Build;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.data.local.storage.LyricSdCard;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.local.storage.LyricSdCard. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports